contracts/access/IAccessControlAdminRules.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.9.0 (access/IAccessControlAdminRules.sol)
+
+pragma solidity ^0.8.0;
+
+import "./IAccessControl.sol";
+
+/**
+ * @dev External interface of AccessControladminRules declared to support ERC165 detection.
+ *
+ * _Available since v4.9._
+ */
+interface IAccessControlAdminRules is IAccessControl {
```

Comment:
Definitely, I don't like having the docs split across functions and only referenced with `@dev See {...}`.

> which would not work with just AccessControl.

Are you referring to `type(IAccessControlAdminRules)` and `IAccessControlAdminRules` instead? Otherwise, can you clarify? I don't follow _what_ would not work.